client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -0,0 +1,230 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import StatsList from '../stats-list';
+import SelectDropdown from 'components/select-dropdown';
+import toggle from '../mixin-toggle';
+import skeleton from '../mixin-skeleton';
+import ErrorPanel from '../stats-error';
+import Pagination from '../pagination';
+import analytics from 'analytics';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'StatsFollowers',
+
+	mixins: [ toggle( 'FollowersPage' ), skeleton( 'data' ) ],
+
+	data( nextProps ) {
+		const props = nextProps || this.props;
+		return props.followersList.response.data;
+	},
+
+	getInitialState() {
+		return {
+			activeFilter: 'wpcom-followers',
+			noData: this.props.followersList.isEmpty( 'subscribers' )
+		};
+	},
+
+	componentWillReceiveProps( nextProps ) {
+		this.setState( {
+			noData: nextProps.followersList.isEmpty( 'subscribers' )
+		} );
+	},
```

Comment:
This seems a bit dubious to me since I can't find anywhere that we actually use this state value. In `render()` it uses the `isEmpty` function call directly, which makes more sense to me than storing it in `state`